hs_discover/public/map.js

```diff
@@ -0,0 +1,173 @@
+((exports) => {
+  const mapDefaultZoom = 4;
+  // eslint-disable-next-line no-unused-vars
+  let googMarkers = [];
+  let pois = [];
+  let markerCluster;
+
+  const deleteMarkers = () => {
+    googMarkers.forEach((marker) => {
+      marker.setMap(null);
+    });
+    googMarkers = [];
+    if (markerCluster) {
+      markerCluster.clearMarkers();
+    }
+  };
+
+  const createSearcher = () => {
+    const searchBox = new google.maps.places.SearchBox(document.getElementById('map-search')); // eslint-disable-line
+    exports.map.controls[google.maps.ControlPosition.TOP_CENTER].push(document.getElementById('map-search')); // eslint-disable-line
+    // Bias the SearchBox results towards current map's viewport.
+    exports.map.addListener('bounds_changed', () => {
+      searchBox.setBounds(exports.map.getBounds());
+    });
+
+    let markers = [];
+    searchBox.addListener('places_changed', () => {
+      const places = searchBox.getPlaces();
+
+      if (places.length === 0) {
+        return;
+      }
+
+      markers.forEach((marker) => {
+        marker.setMap(null);
+      });
+      markers = [];
+
+      const bounds = new google.maps.LatLngBounds(); // eslint-disable-line
+      places.forEach((place) => {
+        if (!place.geometry) {
+          console.log('Returned place contains no geometry');
+          return;
+        }
+        // const icon = {
```

Comment:
Looks to be quite a bit of commented out code, please make a pass to clean it up in this file.

Reply:
Commit 70be255c5